### PR TITLE
plugin/log: remove need to specify stdout

### PR DESCRIPTION
**Pull request by nchrisdk**
### 1. What does this pull request do?
Clean up of log plugin to remove need to specify stdout in Corefile

### 2. Which issues (if any) are related?
#1218

### 3. Which documentation changes (if any) need to be made?
README.md of log plugin has been changed to match new format

### CODE REVIEW

File: plugin/log/README.md [hunk 1]
```diff
@@ -10,14 +10,10 @@ log
 
 * With no arguments, a query log entry is written to *stdout* in the common log format for all requests
 
-~~~ txt
-log [stdout]
-~~~
-
```
Comment (miekg) [inline plugin/log/README.md:16]: I would leave this in, but drop the `[stdout]` bit

File: plugin/log/setup_test.go [hunk 2]
```diff
@@ -18,47 +18,43 @@ func TestLogParse(t *testing.T) {
 			NameScope: ".",
 			Format:    DefaultLogFormat,
 		}}},
-		{`log example.org stdout`, false, []Rule{{
+		{`log example.org.`, false, []Rule{{
```
Comment (miekg) [inline plugin/log/setup_test.go]: leaving out the suffix `.` on `example.org` was actually part of the test.

File: plugin/log/setup_test.go [hunk 3]
```diff
@@ -18,47 +18,43 @@ func TestLogParse(t *testing.T) {
...
-			NameScope: "example.org.",
-			Format:    DefaultLogFormat,
-		}}},
-		{`log example.org stdout {common}`, false, []Rule{{
+		{`log example.org. {common}`, false, []Rule{{
 			NameScope: "example.org.",
 			Format:    CommonLogFormat,
 		}}},
-		{`log example.org stdout {combined}`, false, []Rule{{
+		{`log example.org. {combined}`, false, []Rule{{
```
Comment (miekg) [inline plugin/log/setup_test.go]: here too: don't add the dot

**Review by miekg — CHANGES_REQUESTED** [review]
looks good, still some (small) comments.

Thanks for the quick turn around!

File: plugin/log/README.md (hunk 1 above)
Comment (nchrisdk) [reply plugin/log/README.md:16]: Right but then the statement becomes the same as is on line 7-9...

Comment (miekg) [reply plugin/log/README.md:16]: /me checks out PR. Ah yes, you're richt.

Comment (nchrisdk) [reply plugin/log/README.md:16]: =)

**Review by miekg — APPROVED** [review]

**Pull request merged**